### PR TITLE
#491 issue of the tagline getting highlighted solved. 

### PR DESCRIPTION
**Pull request by namrataaaa**
As mentioned in issue #491 the tagline/agenda was also getting selected while hovering over the logo or the main title. This has been resolved.

Checklist
- [ x ] I followed the indications in the [CONTRIBUTING](../CONTRIBUTING.md)
- [ x ] The documentation related to the proposed change has been updated accordingly (also comments in code).
- [ x ] Have you written new tests for your core changes, as applicable?
- [ x ] Have you successfully ran tests with your changes locally?
- [ x ] Ready for review! :rocket:

### CODE REVIEW

**Comment by bfabio** [comment]
@namrataaaa Thanks for the PR :+1: Can you please rebase it on master?

**Comment by namrataaaa** [comment]
> @namrataaaa Thanks for the PR 👍 Can you please rebase it on master?

I rebased it on master. Please have a look and let me know if I need to make any more changes.

File: _includes/header.html [hunk 1]
```diff
@@ -61,10 +61,10 @@ <h1 class="Header-titleLink">
           <a href="{{ '/' | relative_url }}">{{ t.site_title | default: site.title | escape }}
             {% if site.pill != "" and site.pill != nil %}
             <span class="badge badge-pill badge-primary primary-bg-a7 d-none d-sm-inline-block">{{ site.pill }}</span>{%
-                endif %}
-            <span
-              class="d-none d-lg-block Header-subtitle">{{ t.site_subtitle | default: site.subtitle | escape }}</span>
+                endif %>
```
Comment (bfabio) [inline _includes/header.html:64]:
```suggestion
                endif %}
```

**Review by bfabio — CHANGES_REQUESTED** [review]

**Comment by namrataaaa** [comment]
Hi, @bfabio I made the changes as requested. Please have a look and let me know if everything is okay.

**Review by bfabio — CHANGES_REQUESTED** [review]
Hi @namrataaaa, there's something wrong with the header, you can see the preview here: https://deploy-preview-765--developers-italia.netlify.app/

**Comment by namrataaaa** [comment]
@bfabio I apologize for my mistake. It has now been fixed. Have a look please.

File: assets/scss/main.scss [hunk 2]
```diff
@@ -382,11 +382,13 @@ body {
             @include bootstrap.media-breakpoint-up(md) {
                 font-size: 2rem;
             }
-            .Header-subtitle {
+
```
Comment (bfabio) [inline assets/scss/main.scss:385]:
```suggestion
```

File: assets/scss/main.scss [hunk 3]
```diff
@@ -382,11 +382,13 @@ body {
             @include bootstrap.media-breakpoint-up(md) {
                 font-size: 2rem;
             }
-            .Header-subtitle {
+            
+        }
+        .Header-subtitle {
+                color: #fff;
```
Comment (bfabio) [inline assets/scss/main.scss:388]: Let's use 4 spaces for indentation.

File: assets/scss/main.scss [hunk 4]
```diff
@@ -382,11 +382,13 @@ body {
...
                 font-size: 1rem;
                 font-weight: 300;
             }
```
Comment (bfabio) [inline assets/scss/main.scss:391]:
```suggestion
        }
```

**Review by bfabio — CHANGES_REQUESTED** [review]
Almost there :wink:

File: assets/scss/main.scss (hunk 3 above)
Comment (bfabio) [reply assets/scss/main.scss:387]: @namrataaaa  `.Header-subtitle`'s rules are still 8 spaces indented.

**Review by bfabio — APPROVED** [review]

**Comment by bfabio** [comment]
@namrataaaa Thanks for your work, merged :rocket:

**Comment by namrataaaa** [comment]
> @namrataaaa Thanks for your work, merged 🚀

Thank you too. I learnt a lot and had a great experience. 😄

**Pull request merged**